packages/cli/templates/webcomponents/igc-ts/index.ts

```diff
@@ -0,0 +1,17 @@
+import { BaseProjectLibrary } from "@igniteui/cli-core";
+
+class IgcWebComponentsProjectLibrary extends BaseProjectLibrary {
+	constructor() {
+		super(__dirname);
+		this.name = "Ignite UI for Web Components";
+		this.projectType = "igc-ts";
+		this.themes = ["default"];
+
+		const groups = require("./groups.json");
+		// tslint:disable-next-line:forin
```

Comment:
Remove tslint (// tslint:disable-next-line:forin) comment and actually fix the stuff.
It actually applies to all instances of this comment across projects and can be fixed with the same snippet below.
Can include a check groups.hasOwnProperty(key). For example:

		for (const key in groups) {
			if (groups.hasOwnProperty(key)) {
				this.groupDescriptions.set(key, groups[key]);
			}
		}